yarn.lock

```diff
@@ -36,9 +36,9 @@
     "@jridgewell/trace-mapping" "^0.3.9"
 
 "@jridgewell/resolve-uri@^3.0.3":
-  version "3.0.8"
-  resolved "https://registry.yarnpkg.com/@jridgewell/resolve-uri/-/resolve-uri-3.0.8.tgz#687cc2bbf243f4e9a868ecf2262318e2658873a1"
-  integrity sha512-YK5G9LaddzGbcucK4c8h5tWFmMPBvRZ/uyWmN1/SbBdIvqGUdWGkJ5BAaccgs6XbzVLsqbPJrBSFwKv3kT9i7w==
+  version "3.1.0"
```

Comment:
Should yarn.lock be checked in at all? Should we be ignoring it in .gitignore?

Reply:
Yes it should be checked in. The `yarn.lock` ensures that no matter who pulls this code when they run `yarn install` everybody would get the same version of the packages. In a handful of places in the `package.json` we need to use `^VERSION` meaning anything above and including that specified version can be used.

Without the `yarn.lock` the items that don't have a strict version can change depending on when the `yarn install` command is ran. Eg on 5/6 John Smith cloned the repo, ran `yarn install` and got version `7.1.3` of `@types/glob`. However, later that day version `7.1.4` was released to npmjs. On 5/7 Jane Doe cloned the repo, ran `yarn install` and got version `7.1.4` of `@types/glob` but for this individual something broke in `daffodil-vscode` from the changes made in `@types/glob` between `7.1.3` and `7.1.4`.